config/locales/en.yml

```diff
@@ -76,3 +78,38 @@ en:
       - "Aurora O'Connor"
       - Karina Patterson
       - Zack Suranofsky
```

Comment:
Sidenote, this is outdated quite a bit now

Reply:
I''ll do another PR, I can't remember whether @sherson still has a "things to do when a new staff member is hired" document, but this should go on there.